Disable gamma setting. We set this early at boot and we don't want X to mess with our settings.
diff -paur xf86-video-intel-2.14.0.orig/src/intel_display.c xf86-video-intel-2.14.0.work/src/intel_display.c
--- xf86-video-intel-2.14.0.orig/src/intel_display.c	2011-01-10 15:02:16.534513000 -0800
+++ xf86-video-intel-2.14.0.work/src/intel_display.c	2011-04-18 13:40:46.586165000 -0700
@@ -617,6 +617,7 @@ static void
 intel_crtc_gamma_set(xf86CrtcPtr crtc,
 		       CARD16 *red, CARD16 *green, CARD16 *blue, int size)
 {
+	return;
 	struct intel_crtc *intel_crtc = crtc->driver_private;
 	struct intel_mode *mode = intel_crtc->mode;
 
